### ISSUE TITLE
WxMaxima for Ubuntu does not  hide the cells

### ISSUE THREAD

**Issue by lvch** [andrejv/wxmaxima#469]
Different result of making 'hide' for the set of input cells, if the first line is blank.
WxMaxima for Windows hides the input cells correctly (see screenshots win1.png and win2.png),
however, WxMaxima for Ubuntu does not work correctly (see ununtu1.png and ubuntu2.png).
It does not  hide the cells.

![win1](https://cloud.githubusercontent.com/assets/6746594/7511075/ec967162-f4aa-11e4-8e49-bf008c6233be.png)
![win2](https://cloud.githubusercontent.com/assets/6746594/7511080/f5ce22a2-f4aa-11e4-98fc-56096dc537ac.png)
![ubuntu1](https://cloud.githubusercontent.com/assets/6746594/7511081/f9d21e8a-f4aa-11e4-862f-54ddbc56a2d0.png)
![ubuntu2](https://cloud.githubusercontent.com/assets/6746594/7511082/fbad880c-f4aa-11e4-98b0-b68f2ddf58eb.png)

**Comment by gunterkoenigsmann** [comment]
Where did you get wxMaxima from? From the package repository or from a ppa? Seems I cannot reproduce the problem on my Ubuntu box.

**Comment by lvch** [comment]
Earlier, I took wxmaxima from here:
https://launchpad.net/~blahota/+archive/ubuntu/wxmaxima

Because there are a bit obsolete version, I installed WxMaxima, built 
myself from source code.

I suppose, you do not make empty the very first lines.
I send you the example with the first three _empty_ lines for the 
recent version of wxmaxima.

If you cannot reproduce this, I am ready to install wxmaximaa from any 
ppa that you  trust.
